app/src/main/java/com/example/game/fragments/CommunityProfileFragment.java

```diff
@@ -0,0 +1,35 @@
+package com.example.game.fragments;
+
+import android.util.Log;
+
+import com.example.game.R;
+import com.example.game.models.Community;
+import com.example.game.models.Subscription;
+import com.parse.ParseQuery;
+import com.parse.ParseUser;
+
+public class CommunityProfileFragment extends CommunitySearchFragment {
+    private static final String TAG = "CommunityPpFragment";
```

Comment:
typo?

Reply:
No, I had a longer tag so I shortened. It was more than the required 23 characters.